plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.tsx

```diff
@@ -20,14 +20,25 @@ import { Chip, Typography } from '@material-ui/core';
 import React from 'react';
 import { catalogImportApiRef } from '../../api';
 
-export const ImportInfoCard = () => {
+type Props = {
+  exampleEntityFilepath?: string;
+  exampleRepositoryPath?: string;
+};
+
+export const ImportInfoCard = ({
+  exampleEntityFilepath = 'https://github.com/backstage/backstage/blob/master/catalog-info.yaml',
+  exampleRepositoryPath = 'https://github.com/backstage/backstage',
```

Comment:
Yeah these props are probably fine - they stay local to the place where they are actually needed. So what remains then is to make them easy for the user to set. What if they could be set on `DefaultImportPage`? That's already exposed publicly, and it can be responsible for passing them down. That means that the user would do like

```tsx
<Route path="/catalog-import" element={
  <CatalogImportPage>
    <DefaultImportPage exampleEntityFilepath="..." />
  </CatalogImportPage>
} />
```

Reply:
This would then also be where the user could pass in a branch name - making the config parameter for that unnecessary. I think it might not be particularly common to want to tweak, and its scope is really limited to within this component.

Reply:
I would love to have something like this! However as I see it leaves two problems:
- How to pass `exampleEntityFilepath` to `StepInitAnalyzeUrl`. The best I've come up with is to provide it via `generateStepper` to `ImportStepper`. Please see my other comment about this.
- Branch name is used in `CatalogImportClient` and I wouldn't want to create my own implementation just to patch this branch name in couple of places. Thus I thought having it as a config option would be the best option.